app/frontend/src/utils.js

```diff
@@ -0,0 +1,124 @@
+
+export const users = [
```

Comment:
is that suppose to get in the master?

Reply:
currently yes - its the only way we can test the screens - as there is no data.
it will be deleted at the end